ops/rollup.json

```diff
@@ -11,11 +11,17 @@
     "l2_time": 1647573629
   },
 
-  "block_time": 1,
+  "block_time": 2,
 
-  "max_sequencer_time_diff": 10,
+  "max_sequencer_drift": 10,
 
-  "seq_window_size": 64,
+  "seq_window_size": 2,
 
-  "l1_chain_id": 900
+  "l1_chain_id": 900,
+
+  "fee_recipient_address":  "0xff00000000000000000000000000000000000002",
+
+  "batch_inbox_address":  "0xff00000000000000000000000000000000000002",
```

Comment:
It's fine if they are the same, but note that all L2 fees will effectively get burned by sending them to `0xff00000000000000000000000000000000000002`. That's fine for testing, but can't have that in production :sweat_smile: Copy-paste error or intentional?

Reply:
I meant to make it `0xff00000000000000000000000000000000000001` like in the E2E tests, but on second thought I'll make this a Hardhat default address so the fees can be recovered in the itests.